crates/ra_hir_def/src/adt.rs

```diff
@@ -57,18 +60,51 @@ pub struct FieldData {
     pub visibility: RawVisibility,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ReprKind {
+    Packed,
+    Other,
+}
+
+fn repr_from_value<T: AttrsOwner>(ast: &T) -> Option<ReprKind> {
```

Comment:
This should be migrated to use semantic `Attrs` rather the syntactic ones.

Reply:
Done